src/plugins/intel_cpu/src/nodes/conv.cpp

```diff
@@ -347,6 +347,9 @@ const std::vector<impl_desc_type>& Convolution::getDefaultImplPriority() {
         impl_desc_type::jit_avx2_dw,
         impl_desc_type::jit_avx2_1x1,
         impl_desc_type::jit_avx2,
+        //Lower down brgemm avx2 primitive priority than jit avx2 for perf issue.
```

Comment:
If we have ticket to track brgemm vs jit perf issue I would ask you to add the number here.
Same for other nodes

Reply:
Added.